ddtrace/monkey.py

```diff
@@ -219,14 +228,22 @@ def patch_module(module, raise_errors=True):
         return False
 
 
+@deprecated(
+    message="This function will be removed.",
+    version="1.0.0",
+)
 def get_patched_modules():
```

Comment:
we should keep the `# type: () -> List[str]` comment here just like it was before (like how it is for `_get_patched_modules` right now)